### PR TITLE
feat: preview image

### PR DESCRIPTION
**Pull request by crazyair**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [x] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     图片预览      |
| 🇨🇳 Chinese |    图片预览       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
这是啥？

**Comment by crazyair** [comment]
> 这是啥？

我测试下加 header 能让 diff 图片使用浏览器预览吗 https://help.aliyun.com/zh/oss/developer-reference/upload-a-local-file?spm=a2c4g.11186623.0.i28

File: scripts/visual-regression/upload.js [hunk 1]
```diff
@@ -92,6 +92,8 @@ async function uploadFile(client, filePath, refValue) {
     'x-oss-object-acl': 'public-read',
     // https://help.aliyun.com/zh/oss/developer-reference/prevent-objects-from-being-overwritten-by-objects-that-have-the-same-names-3
     'x-oss-forbid-overwrite': 'false',
+    'Content-Disposition': 'inline',
```
Comment (fengmk2) [inline scripts/visual-regression/upload.js:95]: https://www.alibabacloud.com/help/zh/oss/user-guide/manage-object-metadata-10#concept-lkf-swy-5db

x-oss-force-download 可以不加

Comment (crazyair) [reply scripts/visual-regression/upload.js:95]: 一开始没加 download，不行，后面才加的

**Review by MadCcc — APPROVED** [review]

**Review by li-jia-nan — APPROVED** [review]

**Pull request merged**